<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="aditya-BBR-WAX9" failures="0" tests="6" name="Sai_Roshini.To_check_ssl" time="0.942" errors="1" timestamp="2023-07-24T21:28:00 IST" skipped="5">
  <testcase classname="Sai_Roshini.To_check_ssl" name="pre_enq_upto_GetScore" time="0.942">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.manage()&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.manage()" because "this.driver" is null
at Sai_Roshini.To_check_ssl.pre_enq_upto_GetScore(To_check_ssl.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- pre_enq_upto_GetScore -->
  <system-out/>
  <testcase classname="Sai_Roshini.To_check_ssl" name="delete_process_enquiry">
    <skipped/>
  </testcase> <!-- delete_process_enquiry -->
  <system-out/>
  <testcase classname="Sai_Roshini.To_check_ssl" name="delete_process_location_and_pre_enq">
    <skipped/>
  </testcase> <!-- delete_process_location_and_pre_enq -->
  <system-out/>
  <testcase classname="Sai_Roshini.To_check_ssl" name="delete_process_bank">
    <skipped/>
  </testcase> <!-- delete_process_bank -->
  <system-out/>
  <testcase classname="Sai_Roshini.To_check_ssl" name="delete_process_loans">
    <skipped/>
  </testcase> <!-- delete_process_loans -->
  <system-out/>
  <testcase classname="Sai_Roshini.To_check_ssl" name="delete_client_master">
    <skipped/>
  </testcase> <!-- delete_client_master -->
  <system-out/>
</testsuite> <!-- Sai_Roshini.To_check_ssl -->
